### PR TITLE
CHROMEOS config/docker: clean up cros-sdk Dockerfile

### PR DESCRIPTION
**Pull request by gctucker**
Clean up the Dockerfile for the cros-sdk image:

* sort package names alphabetically
* split long lines and long commands
* fix indentation issue with mixed tabs and spaces
* rename "user" to "cros" user

Signed-off-by: Guillaume Tucker <guillaume.tucker@collabora.com>

### CODE REVIEW

**Comment by nuclearcat** [comment]
It will require also tiny fix in build_board.sh since username is being changed:
```
diff --git a/config/rootfs/chromiumos/scripts/build_board.sh b/config/rootfs/chromiumos/scripts/build_board.sh
index 392d968..90ba845 100755
--- a/config/rootfs/chromiumos/scripts/build_board.sh
+++ b/config/rootfs/chromiumos/scripts/build_board.sh
@@ -19,7 +19,7 @@ trap cleanup EXIT
 
 echo "Preparing environment, branch ${BRANCH}"
 sudo mkdir chromiumos-sdk
-sudo chown user chromiumos-sdk
+sudo chown ${USERNAME} chromiumos-sdk
 cd chromiumos-sdk
 git config --global user.email "bot@kernelci.org"
 git config --global user.name "KernelCI Bot"
```

**Comment by gctucker** [comment]
> It will require also tiny fix in build_board.sh since username is being changed:
> 
> ```
> diff --git a/config/rootfs/chromiumos/scripts/build_board.sh b/config/rootfs/chromiumos/scripts/build_board.sh
> index 392d968..90ba845 100755
> --- a/config/rootfs/chromiumos/scripts/build_board.sh
> +++ b/config/rootfs/chromiumos/scripts/build_board.sh
> @@ -19,7 +19,7 @@ trap cleanup EXIT
>  
>  echo "Preparing environment, branch ${BRANCH}"
>  sudo mkdir chromiumos-sdk
> -sudo chown user chromiumos-sdk
> +sudo chown ${USERNAME} chromiumos-sdk
>  cd chromiumos-sdk
>  git config --global user.email "bot@kernelci.org"
>  git config --global user.name "KernelCI Bot"
> ```

Fixed in #1297

**Review by nuclearcat — CHANGES_REQUESTED** [review]

File: config/docker/cros-sdk/Dockerfile [hunk 1]
```diff
@@ -5,33 +5,35 @@ MAINTAINER "KernelCI TSC" <kernelci-tsc@groups.io>
...
-	python3 \
-	curl \
-	wget \
-	ssh \
-    && apt-get clean \
+        python3 \
+        sudo \
+        ssh \
+        wget
+RUN get clean \
```
[the remark this answers is not in the document]
Comment (gctucker) [reply config/docker/cros-sdk/Dockerfile:17]: Erm, yes. Not sure how I got that there as I'm pretty sure I built it locally.

[the remark this answers is not in the document]
Comment (gctucker) [reply config/docker/cros-sdk/Dockerfile]: Fixed

File: config/docker/cros-sdk/Dockerfile [hunk 2]
```diff
@@ -5,33 +5,35 @@ MAINTAINER "KernelCI TSC" <kernelci-tsc@groups.io>
...
     && apt-get autoremove \
     && rm -rf /var/lib/apt/lists/*
 
-RUN useradd -u 996 -ms /bin/sh user && adduser user sudo && echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
-RUN mkdir -p /home/user/chromiumos && chown -R user /home/user/chromiumos
+RUN useradd -u 996 -ms /bin/sh cros
+RUN adduser cros sudo && echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+RUN mkdir -p /home/cros/chromiumos
+RUN chown -R cros /home/user/chromiumos
```
[the remark this answers is not in the document]
Comment (gctucker) [reply config/docker/cros-sdk/Dockerfile]: Fixed

**Review by nuclearcat — APPROVED** [review]

**Pull request merged**